R/confirmatoryfactoranalysis.R

```diff
@@ -898,6 +941,38 @@ confirmatoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
     vi$setData(vidat)
     vi$dependOn(optionsFromObject = jrobject)
   }
+
+
+  # Thresholds
+  if ("|" %in% pei$op) {
+
+    # Manifest variable intercepts
+    jrobject[["Thresholds"]] <- th <- createJaspTable(title = gettext("Thresholds"))
+    if (!is.null(footnote)) th$addFootnote(footnote)
+
+    th$addColumnInfo(name = "lhs",    title  = gettext("Indicator threshold"),  type = "string", combine = TRUE)
+    th$addColumnInfo(name = "est",    title  = gettext("Estimate"),   type = "number", format = "sf:4;dp:3")
```

Comment:
```suggestion
    th$addColumnInfo(name = "est",    title  = gettext("Estimate"),   type = "number")
```

for the other columns as well